Source/Internal/IGListAdapterUpdater+DebugDescription.m

```diff
@@ -51,11 +51,6 @@ @implementation IGListAdapterUpdater (DebugDescription)
     }
     [debug addObject:[NSString stringWithFormat:@"State: %@", stateString]];
 
-    if (self.applyingUpdateData != nil) {
-        [debug addObject:@"Batch update data:"];
-        [debug addObjectsFromArray:IGListDebugIndentedLines([self.applyingUpdateData debugDescriptionLines])];
-    }
```

Comment:
Shoot, I think we actually need this property b/c of the debugger (that's why we added it)... dang. Thanks for taking this on tho! My mistake filing w/out doing a bigger investigation..